build/README.md

```diff
@@ -4,13 +4,14 @@ This directory contains all of the individual IPDK recipes. These can all be
 built using the `ipdk` CLI found here.
 
 ## Install IPDK CLI
+
 To fully use all the features of the IPDK containers and if you want to follow
 the examples, then install the IPDK CLI with the following command executed
 from the directory this README is found in:
 
-```
-./ipdk install
-```
+ ```
```

Comment:
Why do we need spaces in these 3 lines?
even with out these space, "./ipdk install" command in main branch readme shows the same way.


Same comment for all the changes below. Why is the format changed when the o/p of readme looks same.
Is it related to any Markdown versioning?

Reply:
yes these changes are to fix lint check errors. You can check the errors here -  https://github.com/ipdk-io/ipdk/runs/7585562790?check_suite_focus=true

Some example errors are here:
./BUILDING.md:69:1 MD004/ul-style Unordered list style [Expected: asterisk; Actual: dash]
./BUILDING.md:72:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
./BUILDING.md:73:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
./BUILDING.md:73:43 MD034/no-bare-urls Bare URL used [Context: "https://docs.docker.com/networ..."]
./BUILDING.md:74:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
./BUILDING.md:75:1 MD007/ul-indent Unordered list indentation [Expected: 2; Actual: 4]
./BUILDING.md:75:1 MD030/list-marker-space Spaces after list markers [Expected: 1; Actual: 2]

Reply:
mdl checks were recently introduced for md files, so we see the failures now

Reply:
got it, thanks for clarification.